### PR TITLE
Made api route for getting books given a subject

### PR DESCRIPTION
**Pull request by albert99tseng**

### CODE REVIEW

File: backend/api/views.py [hunk 1]
```diff
@@ -36,6 +37,38 @@ def post(self, request, *args, **kwargs):
...
+
+
+# class StandardResultsSetPagination(pagination.PageNumberPagination):
+#     page_size = 2
+#     page_size_query_param = 'page_size'
+#     max_page_size = 100
+
+class BookList(generics.ListCreateAPIView):
+    queryset = Exchange.objects.all()
+    serializer_class = SellSerializer
```
Comment (bkcapretta) [inline backend/api/views.py]: Should you use the Exchange serializer instead here?

File: backend/api/views.py [hunk 2]
```diff
@@ -36,6 +37,38 @@ def post(self, request, *args, **kwargs):
...
+        get_subject = request.GET.get('subject', '')
+        filter_subject = queryset.filter(subject_info = get_subject)
+        # gets all textbooks in that queryset
+
+
+        ### LEFT OFF HERE
+#        books = filter_subject.select_related('book_info')
+#        print(books)
+#        return JsonResponse(TextbookSerializer(books, many=True).data, safe=False)
+        return JsonResponse(SellSerializer(filter_subject, many=True).data, safe=False)
```
Comment (bkcapretta) [inline backend/api/views.py]: Again, would it make more sense to use the Exchange serializer?

File: backend/api/views.py (hunk 1 above)
Comment (bkcapretta) [reply backend/api/views.py]: JK realized the SellSerializer uses the exchange model

File: backend/api/urls.py [hunk 3]
```diff
@@ -22,4 +22,5 @@
     path("exchanges/", ListExchangesRaw.as_view(), name="exchange-list"),
     path("subjects/", SubjectList.as_view(), name="subject-list"),
     path("users/", UserList.as_view(), name="user-list"),
+    path('books/', BookList.as_view(), name="book-list"),
```
Comment (barieom) [inline backend/api/urls.py]: For consistency sakes, use " instead of '

File: backend/api/views.py [hunk 4]
```diff
@@ -36,6 +37,38 @@ def post(self, request, *args, **kwargs):
...
+        # serializer = SellSerializer(queryset, many=True)
+        # return Response(serializer.data)
+
+
+    # filter_backends = (DjangoFilterBackend,)
```
Comment (barieom) [inline backend/api/views.py]: Instead of leaving these as comments, document this in the backend README

**Pull request merged**